src/ode_solver/low_storage_runge_kutta_ode_solver.cpp

```diff
@@ -21,85 +20,68 @@ LowStorageRungeKuttaODESolver<dim,real,n_rk_stages, MeshType>::LowStorageRungeKu
         , beta3(this->ode_param.beta3)
 {}
 
-template <int dim, typename real, int n_rk_stages, typename MeshType>
-void LowStorageRungeKuttaODESolver<dim,real,n_rk_stages, MeshType>::step_in_time (real dt, const bool pseudotime)
+template <int dim, typename real, int n_rk_stages, typename MeshType> 
+void LowStorageRungeKuttaODESolver<dim,real,n_rk_stages, MeshType>::calculate_stages(int i, real /*dt*/, const bool pseudotime)
```

Comment:
The name `i` is quite vague, I would rename it to `istage` everywhere.

Reply:
Done in commit 34113a7